services/mgmt/network/autorust.toml

```diff
@@ -0,0 +1 @@
+tags_allow = ["package-2021-08", "package-2021-05"]
```

Comment:
This is limiting the tags for azure_mgmt_network to two of the most recent stable tags.

Reply:
So no root section is necessary? Should we have one to avoid future issues, or likely not a problem?

FWIW, our autorest tooling makes heavy use (required, really) of configuration so I definitely could see adding more in the future here...assuming we don't eventually adopt autorest instead.